app.js

```diff
@@ -2,17 +2,27 @@ import express from "express";
 import bodyParser from "body-parser";
 import cors from "cors";
 import dotenv from "dotenv";
+import session from 'express-session';
 
 import router from "./src/routes/index.js";
 import connection from "./src/models/db.js";
-import checkJwt from "./src/middleware/auth.middleware.js";
-import {getAuthToken} from './src/controllers/auth.controllers.js'
+import checkGoogleAccessToken from "./src/middleware/auth.middleware.js";
+import { getAuthToken } from "./src/controllers/auth.controllers.js";
+import { googleAuthRedirect, googleAuthCallback } from './src/controllers/googleAuth.controllers.js';
 
 dotenv.config();
 
 const app = express();
 const port = Number(process.env.PORT) || 3000;
 
+// Session middleware
+app.use(session({
+  secret: 'your-secret-key',
```

Comment:
what is this `your-secret-key` and where should it come from?

Reply:
Removed, it will be stored on FrontEnd